src/iwant_bot/start.py

```diff
@@ -1,9 +1,25 @@
+import asyncio
 from aiohttp import web
 from iwant_bot import db
 import json
+from os import getenv
+from re import match
+from aioslacker import Slacker
 
 DB_ACCESS = None
 
+VERIFICATION = getenv('VERIFICATION')
+if VERIFICATION is None:
+    print('Warning: Unknown "Verification Token".')
```

Comment:
It would be nice to give a hint that the verification token is expected to be found in the `VERIFICATION` env variable.